bin/kuzzle

```diff
@@ -40,16 +40,16 @@ program
   .arguments('[name]')
   .description('manage plugins')
   .option('    --list', 'list currently installed plugins')
-  .option('    --install', 'if plugin [name] is provided, install it from --version, --url or --path, otherwise, (re-)install all listed plugins')
-  .option('    --remove', 'removes plugin [name] from Kuzzle')
-  .option('    --activate', 'mark the plugin as "activated" (Kuzzle ignores deactivated plugins)')
-  .option('    --deactivate', 'mark the plugin as "deactivated" (Kuzzle ignores deactivated plugins)')
-  .option('    --importConfig <file>', 'import a configuration from a file for a given plugin')
+  .option('    --install', 'install a new plugin package from npm repository if [name] if provided or from --path or --url options')
```

Comment:
As per https://github.com/kuzzleio/kuzzle/blob/develop/lib/api/core/plugins/packages/pluginPackage.js#L21 I don't see any `packageVersion` option available. AFAIK, only `version` is allowed, but not mandatory.